.gitlab-ci/packet.yml

```diff
@@ -333,3 +333,8 @@ packet_ubuntu18-calico-ha-recover-noquorum:
   variables:
     RECOVER_CONTROL_PLANE_TEST: "true"
     RECOVER_CONTROL_PLANE_TEST_GROUPS: "etcd[1:],kube_control_plane[1:]"
+
+packet_ubuntu22-containerd:
```

Comment:
We can use a exist test like `packet_ubuntu22-calico-aio.yml` and need not to create a new one :-)